src/atoms/Badge.tsx

```diff
@@ -10,11 +11,14 @@ export interface BadgeProps {
   type: BadgeType;
 }
 
-export const Badge: FunctionComponent<BadgeProps> = ({ type, children }) => {
+export const Badge: FunctionComponent<BadgeProps & BoxProps> = ({ type, children, ...props }) => {
   const theme = useTheme();
 
   return (
-    <Box backgroundColor={theme.variants.banner[type].color} sx={{ borderRadius: 'banner' }}>
+    <Box
+      backgroundColor={theme.variants.banner[type].color}
+      sx={{ borderRadius: 'banner' }}
+      {...props}>
```

Comment:
the passed through `Badge` `sx` prop will not be taken into account

Reply:
See Slack 😄